extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -389,8 +389,14 @@ public boolean isAnyPropertySet() {
         @ConfigItem(defaultValueDocumentation = "depends on dialect")
         public Optional<Boolean> jdbcWarnings;
 
+        /**
+         * If set, Hibernate will log queries that took more than specified number of milliseconds to execute.
+         */
+        @ConfigItem
+        public Optional<Long> queriesSlowerThanMs;
+
```

Comment:
Also why is it part of the runtime **and** build time config?

Reply:
I just did what what done in the other fields. It didn't make much sense, but the other ones didn't make sense either, so I just went along with it

Reply:
Should I remove it from build time? If so, what about the other properties?

Reply:
I'll have a look this morning and push an additional commit to your PR. There's definitely something fishy I need to figure out.

Reply:
Sounds good

Reply:
I confirm it was a leftover, I dropped it and pushed an additional commit to your PR.

Reply:
Excellent, thanks!